WBooks/src/app/components/AddDeleteButton/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+
+import { ADD, REMOVE } from './constants/texts';
+import styles from './styles';
+
+function AddDeleteButton({ handleRemoveBook, handleAddBook, timeToAdd, styleContainer }) {
+  return (
+    <Fragment>
+      {timeToAdd ? (
```

Comment:
Why instead of having one `TouchableOpacity ` if you are adding and another one if you are deleting, you have only on `TouchableOpacity ` for both cases?